Sources/XiEditor/XiCore.swift

```diff
@@ -34,6 +34,11 @@ protocol XiCore: class {
     func start(plugin: String, in identifier: ViewIdentifier)
     /// Stops the named plugin for the given view.
     func stop(plugin: String, in identifier: ViewIdentifier)
+    /// Asks core to change the language of the buffer associated with the `view_id`.
+    /// If the change succeeds the client will receive a `language_changed` notification.
+    func setLanguage(identifier: ViewIdentifier, languageName: String)
+    /// Saves `frontent_samples` at `destination`.
+    func saveTrace(destination: String, frontendSamples: [[String: AnyObject]])
```

Comment:
I would say, "Requests that core collect and write out traces to a file at `destination`, including the samples from this frontend."

Reply:
Thanks! PR updated